tests/resources/testcases/test_l2fc/scenario.yaml

```diff
@@ -0,0 +1,12 @@
+samples:
+  normal:
+    resolution: 10
+    universe: "0.0 | 0.5 | 1.0"
```

Comment:
For this testcase, I would have expected a continuous range in both samples, and also not consider them as tumor and normal but just a and b or so.

Reply:
good point! I'll change that